sed/plan.sh

```diff
@@ -1,4 +1,4 @@
-pkg_name=sed
+pkg_name="sed"
```

Comment:
We don't need quotes here.

Reply:
This fails shellcheck in CI when I remove the quotes

https://buildkite.com/chef-oss/habitat-sh-core-plans-master-verify/builds/1157#03063088-884b-497b-ba89-c8a3dfa9cddb

```
In sed/plan.sh line 1:
--
  | pkg_name=sed
  | ^-- SC2209: Use var=$(command) to assign output (or quote to assign string).
```

Reply:
I believe this is because `sed` is commonly identified as a command that someone is attempting to run. We should add a shellcheck exception for this plan, on this line.

Reply:
What's the issue with having the quotes?

`git grep pkg_name=\" | wc -l` tells me there are ~95 core plans with quoted pkg_name values; and I'm wondering if I need to address them all at some point - I can file an issue for that and get them updated, but I'd like to track the reason why it's needed.

Reply:
I changed to a shellcheck disable comment, but I'm still curious why we don't use quotes to avoid the concern than shellcheck highlights.

Reply:
Its a false negative in this case. shellcheck thinks you're calling the sed command. There shouldn't be any need to quote a plan name.